code/controllers/subsystem/dbcore.dm

```diff
@@ -77,7 +77,7 @@ SUBSYSTEM_DEF(dbcore)
 	while(length(processing_queries))
 		var/datum/db_query/query = popleft(processing_queries)
 		if(world.time - query.last_activity_time > (5 MINUTES))
-			message_admins("Found undeleted query, please check the server logs and notify coders.")
+			stack_trace("Found undeleted query, check the sql.log for the undeleted query and add a delete call to the query datum.")
```

Comment:
sooooo true
we should just scrape and track the error file instead @Mothblocks 🥺 🙏